src/modules/assessments/components/AssessmentTitle.tsx

```diff
@@ -101,9 +103,9 @@ const AssessmentTitle = ({
         >
           {projectName} ({parseAndFormatDateRange(entryDate, exitDate)})
         </RouterLink>
-      </Typography>
+      </CommonLabeledTextBlock>
```

Comment:
@ttoomey  can you move the new values to here, right below the project name?

There is an alert that shows up for certain annuals to link to related annuals, and it looks strange:
<img width="1201" alt="Screenshot 2024-07-15 at 9 32 56 AM" src="https://github.com/user-attachments/assets/b949165a-05c6-4a9f-85d9-147f32da321c">

Reply:
Also seeing this mismatch in typography, but can be addressed once we have a response https://github.com/open-path/Green-River/issues/6030#issuecomment-2228525042

Reply:
Thanks @gigxz - this should be fixed now. [I made a another issue for the typographic inconsistency ](https://github.com/open-path/Green-River/issues/6371)

Reply:
Thanks @ttoomey !